File: Area/Area.cs
using System;
using System.Linq;

public class Area{
    public static void Main(){
        double[] valores = Console.ReadLine().Split(' ').Select(double.Parse).ToArray();
        Console.WriteLine($"TRIANGULO: {valores[0] * valores[2] / 2:0.000}\nCIRCULO: {3.14159 * Math.Pow(valores[2], 2):0.000}\nTRAPEZIO: {(valores[0] + valores[1]) * valores[2] / 2:0.000}\nQUADRADO: {Math.Pow(valores[1], 2):0.000}\nRETANGULO: {valores[0] * valores[1]:0.000}");
    }
}

File: Area/Area.java
import java.util.Scanner;

public class Area{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        String informacoes = in.nextLine();
        String[] valores = informacoes.split(" ");
        System.out.printf("TRIANGULO: %.3f\nCIRCULO: %.3f\nTRAPEZIO: %.3f\nQUADRADO: %.3f\nRETANGULO: %.3f\n", Double.parseDouble(valores[0]) * Double.parseDouble(valores[2]) / 2, 3.14159 * Math.pow(Double.parseDouble(valores[2]), 2), (Double.parseDouble(valores[0]) + Double.parseDouble(valores[1])) * Double.parseDouble(valores[2]) / 2, Math.pow(Double.parseDouble(valores[1]), 2), Double.parseDouble(valores[0]) * Double.parseDouble(valores[1]));
    }
}